docs/learn/learn-nominator.md

```diff
@@ -185,7 +185,7 @@ to nominate with, in order to receive staking rewards, can change between the er
 - **min-intention-threshold:** minimum stake to declare the intention to nominate. This parameter
   can be updated via on-chain governance and the most recent and up to date version can be found on
   [chain state](https://polkadot.js.org/apps/#/chainstate) (select **state query > staking >
-  minimumNominatorBond**)
+  minimumNominatorBond**) This doesn't show 100, it shows 1,000,000,000,000???
```

Comment:
@DenisonRoad This is the expected output as the result is displayed in Planck's, not DOT.  To convert from Planck to DOT you multiple the input by 0.0000000001.
```
1,000,000,000,000 * 0.0000000001 = 100
```

See [this support article](https://support.polkadot.network/support/solutions/articles/65000168663-how-many-planck-are-in-a-dot-) for more details.

Reply:
I did wonder that ...... thanx